src/components/text_input_wrapper.tsx

```diff
@@ -17,7 +20,7 @@ interface TextInputWrapperProps {
  * @param path The path (consistent with the path provided to the lodash
  * get() method) to the datum within the data store for the TextInput component
  */
-const TextInputWrapper: FC<TextInputWrapperProps> = ({label, path}) => {
+const TextInputWrapper: FC<TextInputWrapperProps> = ({label, path, min=0, max=1000, regexp=/.*/}) => {
```

Comment:
Set the default for `max` to a higher value since this is multi-line text.  Maybe `10240`.